packages/arb-validator-core/ethbridgetest/message_test.go

```diff
@@ -30,230 +41,353 @@ import (
+func TestTransactionMessage(t *testing.T) {
+	msg := message.DeliveredTransaction{
+		Transaction: message.Transaction{
+			Chain:       addr3,
+			To:          addr1,
+			From:        addr2,
+			SequenceNum: big.NewInt(74563),
+			Value:       big.NewInt(89735406),
+			Data:        []byte{65, 23, 68, 87, 12},
+		},
+		BlockNum:  common.NewTimeBlocks(big.NewInt(87962345)),
+		Timestamp: big.NewInt(35463245),
+	}
+	bridgeHash, err := tester.TransactionHash(
+		nil,
+		msg.Chain.ToEthAddress(),
+		msg.To.ToEthAddress(),
+		msg.From.ToEthAddress(),
+		msg.SequenceNum,
+		msg.Value,
+		msg.Data,
+		msg.BlockNum.AsInt(),
+		msg.Timestamp,
+	)
 	if err != nil {
 		t.Fatal(err)
 	}
-	client, err := ethclient.Dial(test.GetEthUrl())
+	if bridgeHash != msg.CommitmentHash().ToEthHash() {
+		t.Error(errHash)
+	}
+
+	messageBridgeHash, err := tester.TransactionMessageHash(
+		nil,
+		msg.Chain.ToEthAddress(),
+		msg.To.ToEthAddress(),
+		msg.From.ToEthAddress(),
+		msg.SequenceNum,
+		msg.Value,
+		msg.Data,
+		msg.BlockNum.AsInt(),
+		msg.Timestamp,
+	)
 	if err != nil {
 		t.Fatal(err)
 	}
-	_, tx, tester, err := messagetester.DeployMessageTester(auth, client)
+
+	if messageBridgeHash != message.DeliveredValue(msg).Hash().ToEthHash() {
+		t.Error(errMsgHash)
+	}
+}
+
+func TestTransactionBatchMessage(t *testing.T) {
+	chain := addr3
+	data := []byte{65, 23, 68, 87, 12}
+	tos := []common.Address{addr1}
+	sequenceNums := []*big.Int{big.NewInt(74563)}
+	values := []*big.Int{big.NewInt(89735406)}
+	dataLengths := []uint32{uint32(len(data))}
+
+	offchainHash := message.OffchainTxHash(
+		chain,
+		tos[0],
+		sequenceNums[0],
+		values[0],
+		data,
+	)
+	messageHash := hashing.SoliditySHA3WithPrefix(offchainHash[:])
+
+	privateKey, err := crypto.HexToECDSA(privateKeyHex)
 	if err != nil {
 		t.Fatal(err)
 	}
-	_, err = ethbridge.WaitForReceiptWithResults(context.Background(), client, auth.From, tx, "DeployMessageTester")
+
+	sig, err := crypto.Sign(messageHash.Bytes(), privateKey)
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	addr1 := common.Address{}
-	addr1[0] = 76
-	addr1[19] = 93
+	msg := message.DeliveredTransactionBatch{
+		TransactionBatch: message.TransactionBatch{
+			Chain:        addr3,
+			Tos:          tos,
+			SequenceNums: sequenceNums,
+			Values:       values,
+			DataLengths:  dataLengths,
+			Data:         data,
+			Signatures:   sig,
+		},
+		BlockNum:  common.NewTimeBlocks(big.NewInt(87962345)),
+		Timestamp: big.NewInt(35463245),
+	}
 
-	addr2 := common.Address{}
-	addr2[0] = 43
-	addr2[19] = 12
+	tx, err := tester.TransactionBatchHash2(
+		auth,
+		msg.Chain.ToEthAddress(),
+		common.AddressArrayToEth(tos),
+		sequenceNums,
+		values,
+		dataLengths,
+		data,
+		sig,
+	)
 
-	addr3 := common.Address{}
-	addr3[0] = 73
-	addr3[19] = 85
+	receipt, err := ethbridge.WaitForReceiptWithResults(context.Background(), client, auth.From, tx, "TransactionBatchHash2")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [line is 125 characters, out of limit 80](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)